lib/internal/freelist.js

```diff
@@ -23,4 +25,14 @@ class FreeList {
   }
 }
 
-module.exports = FreeList;
+function setIsReused(item, reused) {
+  item[is_reused_symbol] = reused;
+  return item;
+}
```

Comment:
Can  you avoid this function call and maybe even the ternary? This will likely inline anyway, but I find putting this in the `alloc` method more readable anyway.

Reply:
Sure.

Reply:
Done.